app/gui/src/main/scala/org/bitcoins/gui/WalletGUI.scala

```diff
@@ -32,6 +32,23 @@ object WalletGUI extends JFXApp {
       }
     )
 
+  private val argsWithIndex = parameters.raw.zipWithIndex
+
+  val rpcPortOpt: Option[Int] = {
```

Comment:
Any way we can factor code like this into a shared trait between everything that `extends App` so we don't have to remember to update different places every time we add an option?